### DISCUSSION TITLE
How do I contribute?

### DISCUSSION THREAD

**Discussion by thetestlabs** [slidevjs/slidev#1642] [Q&A] [answered]
Hi, I've cloned the repository in order to update some documentation. I've created a local branch, committed changes, but when I try to push the branch I get the error:

```
remote: Permission to slidevjs/slidev.git denied to thetestlabs.
```

When I use Gitpod to do the same process I get the error:

```
The remote repository "https://github.com/slidevjs/slidev.git" is not accessible with the current token. Please grant the necessary permissions.
```

I'm keen to contribute, if someone could kindly take the time to explain what I'm missing that would be really appreciated.

**Comment by kermanx** [comment, marked as answer]
Hi, happy to hear you want to contribute!

You can do the following:

1. Fork the repository. For example, to https://github.com/thetestlabs/slidev.
![image](https://github.com/slidevjs/slidev/assets/63178754/88f30205-bfb1-4a58-be9e-2d697b46bb9b)

2. In you local repo, run the following command to change the remote from `slidevjs/slidev` to `thetestlabs/slidev`.
```sh
git remote set-url origin https://github.com/thetestlabs/slidev.git
```

3. Push the branch to the new origin.

4. Go to https://github.com/slidevjs/slidev/pulls, there should be a "Compare & Pull Request" button, where you can create a PR.

**Reply by thetestlabs** [reply]
Thanks for the quick reply!
